src/settings/LoanHistory/LoanHistoryForm.css

```diff
@@ -0,0 +1,16 @@
+.intervalSelector {
+  display: -webkit-inline-box;
+  margin: 0;
+}
+
+.paddingLeft0 {
+  padding-left: 0;
+}
+
+.error {
+  color: #900;
```

Comment:
Use variable from https://github.com/folio-org/stripes-components/blob/master/lib/variables.css#L22.

You may need to add at the top of the file
`@import "@folio/stripes-components/lib/variables.css";`